docs/ovms_quickstart.md

```diff
@@ -0,0 +1,101 @@
+
+Let's use the face detection model available under the following [link](https://download.01.org/opencv/2020/openvinotoolkit/2020.2/open_model_zoo/models_bin/3/face-detection-retail-0004/FP32/). 
+
+Then download both components of the model to the `model` directory (for example using curl):
+```
+curl --create-dirs https://download.01.org/opencv/2020/openvinotoolkit/2020.2/open_model_zoo/models_bin/3/face-detection-retail-0004/FP32/face-detection-retail-0004.xml https://download.01.org/opencv/2020/openvinotoolkit/2020.2/open_model_zoo/models_bin/3/face-detection-retail-0004/FP32/face-detection-retail-0004.bin -o model/face-detection-retail-0004.xml -o model/face-detection-retail-0004.bin
+```
+
+### Start the Model Server container
+
+To start the Model Server container use the following command:
+
+```bash
+docker run -d -v <folder_with_downloaded_model>:/models/face-detection/1 -e LOG_LEVEL=DEBUG -p 9000:9000 openvino/ubuntu18_model_server \
+/ie-serving-py/start_server.sh ie_serving model --model_path /models/face-detection --model_name face-detection --port 9000  --shape auto
+```
+
+Model Server expects models in the predefined structure of folders - that is why the folder with downloaded models is mounted as `/models/face-detection/1`. In our case this expected structure is as follows:
+
+```bash
+models/
+└── face-detection
+    └── 1
+        ├── face-detection-0106.bin
+        └── face-detection-0106.xml
+
+``` 
+
+More about this structure and about how to deploy more than one model at the time can be found [here](./docker_container.md#preparing-the-models) and [here](./docker_container.md#starting-docker-container-with-a-configuration-file).
+
+### Download the example client script
+
+The easiest way to access Model Server is using prepared client scripts. In our case we need to get a script that performs a face detection - like [this](../example_client/face_detection.py).
+
+Together with this script we need to download a requirements.txt file with a list of libraries required to execute the script - this file is located [here](../example_client/client_requirements.txt) and the [file](https://github.com/openvinotoolkit/model_server/blob/master/example_client/client_utils.py) with some additional functions used by the client.
+
+To download all this components we will use curl:
+
+```
+curl https://raw.githubusercontent.com/openvinotoolkit/model_server/master/example_client/client_utils.py -o client_utils.py https://raw.githubusercontent.com/openvinotoolkit/model_server/master/example_client/face_detection.py -o face_detection.py  https://raw.githubusercontent.com/openvinotoolkit/model_server/master/example_client/client_requirements.txt -o client_requirements.txt
+```
+
+More details about this particular script can be found [here](../example_client/face_detection.md). There is more scripts client scripts provided for Model Server - the
+full list is [here](../example_client). 
+
+
+### Download the data needed for inference

```

Comment:
Change to `### Download Data for Inference`

Reply:
done